_posts/2024-11-04-tailoring-your-career-quarterly.html

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: Tailoring Your Career Quarterly Through Transparent Relationships With Your Manager
+date: 2024-11-04
+author_name: Irina Kamalova
+author_role: VP Lead Software Engineer @ JPMorgan Chase
```

Comment:
I can see some images are not been used in the blog post, please could you remove it? 
for example: career-2-about.jpg

Reply:
Also some images have around 1.6MB we need to have optimised sizes, not more than 200px in my opinion, what do you think @khairahscorner ?

Reply:
@khairahscorner I run all images through tinypng but forgot to copy over the compressed version of tips.jpg so I"ll do that now. Thanks!

Reply:
@khairahscorner the 'career-2-about.jpg' image was provided but wasn't included in the content document with the copy. Is it meant to be in the layout? If so where should it be inserted? If not I'll remove.

Reply:
Yes, you can remove it if it's not in the contents @gillian-black 
Thank you!🙌